packages/frontend-2/pages/projects/[id]/index.vue

```diff
@@ -304,6 +306,18 @@ const activePageTab = computed({
   }
 })
 
+const shouldShowWorkspaceAlert = computed(
+  () =>
+    isWorkspacesEnabled.value &&
+    isLoggedIn.value &&
+    !project.value?.workspace &&
+    hasRole.value
+)
+
+const disableLegacyMoveProjectButton = computed(
```

Comment:
We should prob use the canMove permission for this, but we can run with this for now and update that afterwards